### PR TITLE
[SPARK] Test Create/Drop Database Command

### PR DESCRIPTION
**Pull request by amaliujia**
**Description of changes**

Test Create/Drop Database Command for Spark.

### CODE REVIEW

File: connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java [hunk 1]
```diff
@@ -35,6 +35,17 @@ public class SparkIntegrationTest extends BaseCRUDTest {
 
   private final File dataDir = new File(System.getProperty("java.io.tmpdir"), "spark_test");
 
+  @Test
+  public void testCreateSchema() throws ApiException {
+    createCommonResources();
+    SparkSession session = createSparkSessionWithCatalogs(CATALOG_NAME);
+    session.sql("CREATE DATABASE my_test_database;").collect();
```
Comment (cloud-fan) [inline connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java:42]:
```suggestion
    session.sql("CREATE DATABASE my_test_database;");
```

File: connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java [hunk 2]
```diff
@@ -35,6 +35,17 @@ public class SparkIntegrationTest extends BaseCRUDTest {
...
+    assertTrue(session.catalog().databaseExists("my_test_database"));
+    session.sql("DROP DATABASE my_test_database;").collect();
```
Comment (cloud-fan) [inline connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java:44]:
```suggestion
    session.sql("DROP DATABASE my_test_database;");
```

File: connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala [hunk 3]
```diff
@@ -246,9 +246,19 @@ private class UCProxy extends TableCatalog with SupportsNamespaces {
...
-
+  override def createNamespace(namespace: Array[String], metadata: util.Map[String, String]): Unit = {
+    checkUnsupportedNestedNamespace(namespace)
+    val createSchema = new CreateSchema()
+    createSchema.setCatalogName(this.name)
+    createSchema.setName(namespace.head)
+    createSchema.setProperties(metadata)
+    schemasApi.createSchema(createSchema)
+  }
   override def alterNamespace(namespace: Array[String], changes: NamespaceChange*): Unit = ???
```
Comment (cloud-fan) [inline connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala:257]:
```suggestion

  override def alterNamespace(namespace: Array[String], changes: NamespaceChange*): Unit = ???
```

File: connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java [hunk 4]
```diff
@@ -39,6 +39,18 @@ public class SparkIntegrationTest extends BaseCRUDTest {
...
   private final File dataDir = new File(System.getProperty("java.io.tmpdir"), "spark_test");
 
+  @Test
+  public void testCreateSchema() throws ApiException {
+    createCommonResources();
+    SparkSession session = createSparkSessionWithCatalogs(CATALOG_NAME);
+    session.catalog().setCurrentCatalog(CATALOG_NAME);
+    session.sql("CREATE DATABASE my_test_database;");
+    assertTrue(session.catalog().databaseExists("my_test_database"));
+    session.sql("DROP DATABASE uc_testcatalog.my_test_database;");
```
Comment (cloud-fan) [inline connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java:49]: let's use `CATALOG_NAME`

File: connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java [hunk 5]
```diff
@@ -39,6 +39,18 @@ public class SparkIntegrationTest extends BaseCRUDTest {
 
   private final File dataDir = new File(System.getProperty("java.io.tmpdir"), "spark_test");
 
+  @Test
+  public void testCreateSchema() throws ApiException {
+    createCommonResources();
+    SparkSession session = createSparkSessionWithCatalogs(CATALOG_NAME);
+    session.catalog().setCurrentCatalog(CATALOG_NAME);
```
Comment (cloud-fan) [inline connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java:46]: does this work with `spark_catalog`?

Comment (amaliujia) [reply connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java:46]: I extended the test suite to also test with `spark_catalog`

**Review by cloud-fan — APPROVED** [review]

**Pull request merged**